style_about.css

```diff
@@ -0,0 +1,692 @@
+    margin-top: 0;
+  }
+
+  .nav-list-desktop {
+    display: none;
+  }
+
+  .upperpart {
+    display: none;
+  }
+
+  hr {
+    display: none;
+  }
+}
+
+@media only screen and (min-width: 768px) {
+  #stickers {
+    display: none;
+  }
+
+  .upperpart {
+    display: flex;
+    flex-direction: row;
+    background-color: #272a31;
+    height: min-content;
+    align-items: center;
+    place-content: flex-end;
+    padding-right: 5%;
+  }
+
+  footer img {
+    padding-left: 150px;
+    width: 60px;
+    height: 60px;
+  }
+
+  .upperpart img {
+    width: 30px;
+    padding-right: 20px;
+  }
+
+  footer h4 {
+    margin-top: 5px;
+    color: white;
+    font-family: "LatoWeb", sans-serif;
+    font-weight: 500;
+  }
+
+  .upperpart h4 {
+    color: white;
+    font-family: "LatoWebThin", sans-serif;
+    font-size: small;
+    margin-right: 25px;
+  }
+
+  .header {
+    background-color: #d3d3d3a8;
+    border-bottom: 1px solid #272a312f;
+  }
+
+  .nav-list {
+    display: none;
+  }
+
+  .close-hamburger {
+    display: none;
+  }
+
+  .nav-list-desktop {
+    padding-left: 0;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    list-style: none;
+    border: 1px solid #272a312f;
+    background-color: white;
+    margin-top: 0;
+  }
+
+  .nav-list-desktop img {
+    width: 50px;
+  }
+
+  .container-logo {
+    padding-left: 15%;
+  }
+
+  .navbar-about {
+    display: none;
+  }
+
+  .menu-links {
+    padding-right: 20px;
+    text-decoration: none;
+    color: black;
+  }
+
+  .list-container {
+    display: flex;
+    align-items: center;
+    gap: 35px;
+    padding-right: 5%;
+  }
+
+  .menu-links-project {
+    text-decoration: none;
+    border: 3px solid #ec5242;
+    height: 30px;
+    margin-right: 20px;
+    width: 100px;
+    text-align: center;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .menu-links-project a {
+    color: #ec5242;
+    font-family: "LatoWebThin", sans-serif;
+    vertical-align: baseline;
+    padding-right: 0;
+    font-size: medium;
+    font-weight: bolder;
+  }
+
+  .header-h2-title {
+    color: #d72311;
+    padding-top: 100px;
+    font-family: "LatoWebThin", sans-serif;
+    font-size: xx-large;
+    display: flex;
+    justify-content: center;
+    font-weight: bold;
+  }
+
+  .header-h1-title {
+    color: #d72311;
+    display: flex;
+    justify-content: center;
+    font-family: "LatoWebBlack", sans-serif;
+    font-size: xxx-large;
+  }
+
+  .header-p {
+    width: 50%;
+    text-align: center;
+    background-color: white;
+    padding-top: 10px;
+    padding-bottom: 10px;
+    border: 3px solid #272a315d;
+    margin: auto;
+  }
+
+  .email {
+    text-align: center;
+    margin-bottom: 150px;
+    margin-top: 40px;
+  }
+
+  footer {
+    margin-top: 100px;
+    background-color: #272a31;
+    display: flex;
+    flex-direction: row;
+    justify-content: left;
+    align-items: center;
+  }
```

Comment:
- [ ] Kindly make sure to center the elements in the footer of the "About page" as shown in the Behance design below:
- [ ] Also please make sure to stick to the original background color which is `#2e2e2e`.
- Behance design:
 ![image](https://github.com/SaveryIV/First-Capstone-Project/assets/104968236/27c6f21a-e14c-414b-bc10-ac57e0264fdf)
- Your design:
  ![image](https://github.com/SaveryIV/First-Capstone-Project/assets/104968236/a86b2872-9285-4178-84d3-2d913b72c243)